node/chainSimulator/components/statusComponents.go

```diff
@@ -20,7 +21,7 @@ type statusComponentsHolder struct {
 }
 
 // CreateStatusComponentsHolder will create a new instance of status components holder
-func CreateStatusComponentsHolder(shardID uint32) (factory.StatusComponentsHolder, error) {
+func CreateStatusComponentsHolder(shardID uint32) (factory.StatusComponentsHandler, error) {
```

Comment:
Please rename to `CreateStatusComponents` and remove the inner closeHandler declaration & usage

Reply:
renamed